.github/workflows/build_deploy.yaml

```diff
@@ -0,0 +1,62 @@
+name: "Dev CI/CD"
+on:
+  push:
+    branches:
+      - "**"
+      - "!main"
+      - "!master"
+    paths:
+      - '**'
+
+jobs:
+  build_fe:
+    if: ${{ github.ref_name != 'develop' }}
+    name: Build frontend
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+      - name: Set Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - name: Install dependencies
+        run: |
+          npm install  
```

Comment:
Bisogna inserire il flag `--legacy-peer-deps` al comando `npm install` in quanto le peerDependencies di mui-italia sono abbastanza outdated e non possiamo "rispettarle" per adesso.
O, in alternativa, si potrebbe creare il file `.npmrc` nella root del progetto con `legacy-peer-deps=true` in modo da ignorare le peerDependencies di default.

Reply:
Grazie! Inseriamo il flag per mantenere traccia più visibile e mandare avanti la PR

Reply:
@Carminepo2 errata corrige abbiamo inserito il file .npmrc poichè il flag sembra non aver sortito effetti

Reply:
Mi sono accorto dopo che ci sono due step con il  comando `npm install`, quindi bisognava inserirlo in entrambi (linea 27 e linea53). Però sì, direi anche meglio con l'`.npmrc` che lo risolve alla radice. Grazie!